server/resolver.ts

```diff
@@ -0,0 +1,15 @@
+import { createResolveHttp, resolveFile } from '@stoplight/json-ref-readers';
+import { Resolver } from '@stoplight/json-ref-resolver';
```

Comment:
@tillig Can't we reuse the exported `httpAndFileResolver` from Spectral?

Reply:
I think this is the sort of thing that you have more context on that @tillig, so if you see an improvement available in Stoplight tooling it's probably free for you to make. 

That said, Maybe we could list some of the less important things and get them improved as a tech debt item after the merge is complete?

Reply:
Agreed, this is one where you'll know better than I would. That said, [you have pretty much this exact thing in your existing extension, too.](https://github.com/stoplightio/vscode-spectral/blob/8b8fd28152d1dbf889d8168fe955736ae4123ee9/src/resolver.ts#L9)